lightkurve/targetpixelfile.py

```diff
@@ -316,13 +316,16 @@ def centroids(self, aperture_mask='pipeline'):
 
         return col_centr, row_centr
 
-    def plot(self, frame=0, cadenceno=None, bkg=False, aperture_mask=None,
-             **kwargs):
+    def plot(self, ax=None, frame=0, cadenceno=None, bkg=False, aperture_mask=None,
+            colorbar=True, mask_color='pink', **kwargs):
         """
         Plot a target pixel file at a given frame (index) or cadence number.
 
         Parameters
         ----------
+        ax : matplotlib.axes._subplots.AxesSubplot
+            A matplotlib axes object to plot into. If no axes is provided,
+            a new one be generated.
```

Comment:
``a new one be generated `` ``--->`` ``a new one will be generated`` (?)

Reply:
👍